libraries/AP_Compass/AP_Compass.cpp

```diff
@@ -1906,17 +1946,17 @@ bool Compass::consistent() const
         const float xy_ang_diff  = acosf(constrain_float(mag_field_xy*primary_mag_field_xy_norm,-1.0f,1.0f));
 
         // check for gross misalignment on all axes
-        if (xyz_ang_diff > AP_COMPASS_MAX_XYZ_ANG_DIFF) {
+        if (xyz_ang_diff > radians(get_ang_diff_xyz_max())) {
```

Comment:
Instead of calling getter functions here can the class use its private member directly? As it does for other private members in this class?

Reply:
Yes, that was the plan initially, but I made the getter functions for higher reachability, in case we need to access these values in other classes. And the reason I used the method instead of the variable is to keep it consistent, such that if these values would be accessed in other classes/modules, the same method would be called throughout the code, keeping the accessifiers for these values consistent everywhere.

Reply:
If we need to access this data outside of this class then we can add the accessors. Adding the functions just in case bloats the header.

Reply:
I have made the desired changes. Thanks! for the feedback